src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -294,11 +294,30 @@
             "type": "boolean",
             "secret": False,
         },
+        {
+            "id": "request_timeout",
+            "label": ("Request Timeout"),
+            "type": "string",
+            "secret": False,
+            "default": "10",
+            "help_text": (
+                "Specify the timeout Ansible should use in requests to"
+                "the host. Defaults to 10s",
+            ),
+        },
     ],
     "required": ["host"],
 }
 
 AAP_INJECTORS = {
+    "extra_vars": {
+        "aap_hostname": "{{host}}",
+        "aap_username": "{{username}}",
+        "aap_password": "{{password}}",
+        "aap_token": "{{oauth_token}}",
+        "aap_request_timeout": "{{request_timeout}}",
+        "aap_validate_certs": "{{verify_ssl}}",
```

Comment:
```suggestion
        "aap_verify_ssl": "{{verify_ssl}}",
```
Hey @sean-m-sullivan ,
This should be updated to be in line with the [proper key](https://github.com/ansible/event-driven-ansible/blob/main/plugins/module_utils/arguments.py#L34)